roles/lib_openshift/src/ansible/oc_secret.py

```diff
@@ -0,0 +1,37 @@
+# pylint: skip-file
+# flake8: noqa
+
+
+def main():
+    '''
+    ansible oc module for managing OpenShift Secrets
+    '''
+
+    module = AnsibleModule(
+        argument_spec=dict(
+            kubeconfig=dict(default='/etc/origin/master/admin.kubeconfig', type='str'),
+            state=dict(default='present', type='str',
+                       choices=['present', 'absent', 'list']),
+            debug=dict(default=False, type='bool'),
+            namespace=dict(default='default', type='str'),
```

Comment:
The fragment banners PR was merged earlier today, you may need to pull and rebase on top of that in the update.